release.sh

```diff
@@ -0,0 +1,351 @@
+  if [ -f "$1/Cargo.lock" ]; then
+    sed -i.release.bak -e "s/$SED_FRIENDLY_LAST_VERSION/$NEW_VERSION/g" "$1/Cargo.lock"
+    rm "$1/Cargo.lock.release.bak"
+  fi
+
+  # 8.9. Commit changes.
+  OLD_REPO=`echo "$1" | cut -d '/' -f 2`
+  git commit -a -m "Release $OLD_REPO $NEW_VERSION"
+}
+
+TARGETS="packages/fxa-auth-db-mysql
+packages/fxa-auth-server
+packages/fxa-content-server
+packages/fxa-customs-server
+packages/fxa-email-event-proxy
+packages/fxa-email-service
+packages/fxa-profile-server"
+
+while read -r TARGET; do
+  bump "$TARGET"
+done <<< "$TARGETS"
+
+# 9. Create a tag.
+git tag -a "$NEW_TAG" -m "$BUILD_TYPE release $NEW_VERSION"
+
+PRIVATE_REMOTE=`git remote -v | grep "mozilla/fxa-private.git" | cut -f 1 | head -n 1`
+
+if [ "$PRIVATE_REMOTE" = "" ]; then
+  echo "Warning: No private remote detected, creating one."
+  git remote add private git@github.com:mozilla/fxa-private.git
+  PRIVATE_REMOTE=private
+fi
+
+PRIVATE_BRANCH="$TRAIN_BRANCH-private"
+PRIVATE_REMOTE_BRANCH="$PRIVATE_REMOTE/$PRIVATE_BRANCH"
+
+# 10. Create or checkout the private train branch.
+PRIVATE_BRANCH_EXISTS=`git branch | awk '{$1=$1};1' | grep "^$PRIVATE_BRANCH\$"` || true
+if [ "$PRIVATE_BRANCH_EXISTS" = "" ]; then
+  git fetch "$PRIVATE_REMOTE" "$PRIVATE_BRANCH" || true
+
+  PRIVATE_REMOTE_BRANCH_EXISTS=`git branch -r | awk '{$1=$1};1' | grep "^$PRIVATE_REMOTE_BRANCH\$"` || true
+  if [ "$PRIVATE_REMOTE_BRANCH_EXISTS" = "" ]; then
+    git checkout --no-track -b "$PRIVATE_BRANCH" "$PRIVATE_REMOTE/master"
+  else
+    git checkout --track -b "$PRIVATE_BRANCH" "$PRIVATE_REMOTE_BRANCH"
+  fi
+else
+  git checkout "$PRIVATE_BRANCH"
+  git pull "$PRIVATE_REMOTE" "$PRIVATE_BRANCH" || true
+fi
+
+# 11. Merge train branch into the private train branch.
+git merge "$TRAIN_BRANCH" -m "Merge $TRAIN_BRANCH into $PRIVATE_BRANCH"
+
+# 12. Create a private tag.
+PRIVATE_TAG="$NEW_TAG-private"
+PRIVATE_VERSION="$NEW_VERSION-private"
+git tag -a "$PRIVATE_TAG" -m "$BUILD_TYPE release $PRIVATE_VERSION"
+
+# 13. Return to the original branch.
+git checkout "$CURRENT_BRANCH"
+
+# 14. Tell the user what we did.
+echo
+echo "Success! The release has been tagged locally but it hasn't been pushed."
+echo "Before pushing, you should check that the changes appear to be sane."
+echo "At the very least, eyeball the diffs and git log."
+echo "If you're feeling particularly vigilant, you may want to run some of the tests and linters too."
+echo
+echo "Branches:"
+echo
+echo "  $TRAIN_BRANCH"
+echo "  $PRIVATE_BRANCH"
+echo
+echo "Tags:"
+echo
+echo "  $NEW_TAG"
+echo "  $PRIVATE_TAG"
+echo
+echo "When you're ready to push, paste the following lines into your terminal:"
+echo
+echo "git push origin $TRAIN_BRANCH"
+echo "git push origin $NEW_TAG"
+echo "git push $PRIVATE_REMOTE $PRIVATE_BRANCH"
+echo "git push $PRIVATE_REMOTE $PRIVATE_TAG"
+echo
+echo "After that, you must open pull requests in both the public and private repos to merge the changes back to master."
```

Comment:
Could we use [git request-pull](https://git-scm.com/docs/git-request-pull). If so maybe we add that as command to paste?

Reply:
`request-pull` is a new one on me, thanks. Not sure I fully understand the man page though, does it physically create the PR on GitHub? Can we customise the PR description etc to include `@mozilla/fxa-devs r?` and so on?

Reply:
Bleh..I am not sure how `git request-pull` is working either. Maybe it would be suffice to print out a link that will at least a webpage. Ex. `https://github.com/mozilla/fxa/compare/train-xxx?expand=1`